internal/pkg/log/level.go

```diff
@@ -0,0 +1,104 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+)
+
+type Level string
```

Comment:
This needs to be in the exported `auto` package so users can reference it.

Reply:
This lives in an internal package currently. It needs to be moved to the user accessible `go.opentelemetry.io/auto` package instead.

Reply:
@MrAlias nice, changed to `auto` package the level